src/collective/fingerpointing/utils.py

```diff
@@ -29,13 +46,27 @@ def get_request_information():
     return user_id, ip
 
 
+def isCompressedFile(filename):
```

Comment:
please [don't use `camelCase` for function names](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names).

IMO, it would be way easier just to check if the extension is not one of a compressed file.